src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -44,16 +43,13 @@ public SqliteConnection(string connectionString)
             ConnectionString = connectionString;
         }
 
-        internal virtual Sqlite3Handle DbHandle
-            => _db;
-
         /// <summary>
         /// Gets a handle to underlying database connection.
         /// </summary>
         /// <value>A handle to underlying database connection.</value>
         /// <seealso href="http://sqlite.org/c3ref/sqlite3.html">Database Connection Handle</seealso>
-        public virtual IntPtr Handle
-            => _db?.DangerousGetHandle() ?? IntPtr.Zero;
+        public virtual sqlite3 Handle
```

Comment:
When API check is re-enabled, this will probably fail here. cref https://github.com/aspnet/KoreBuild/pull/163